### DISCUSSION TITLE
Is there a way to get all enclosing features for a latitude / longitude?

### DISCUSSION THREAD

**Discussion by davidbarratt** [osm-search/Nominatim#3102] [Q&A]
I'm looking for a way to get all of the enclosing features for a latitude / longitude.

Basically I'm looking for a similar feature to what is on OSM when you right click on the map and select "Query Features" on one of the queries is the "Enclosing features". I think this currently uses the [Overpass API](https://wiki.openstreetmap.org/wiki/Overpass_API) is there a way to do this with Nominatim?

**Comment by mtmail** [comment]
Nominatim's reverse search algorithm works differently and there's no equivalent to what Overpass API does. Nominatim looks for the nearest named feature (or having a house number or similar), that means a lot of places in OSM data might already be skipped. Nearest might also be 100s of meters away from the search position. Then from the found feature it looks up the address hierarchy. Nominatim will include places that aren't area, e.g. a city node if a city doesn't have boundaries in OSM data yet. In the address hierarchy non-relevant area will also be skipped. For example a tourist attraction in a park will be assigned a street, neighborhood, city but the park itself isn't part of address.
